openslides/assignments/apps.py

```diff
@@ -95,3 +100,21 @@ def get_startup_elements(self):
                 if option:
                     candidates.add(option["user_id"])
     return candidates
+
+
+async def required_users_polls(element: Dict[str, Any]) -> Set[int]:
```

Comment:
Please use options directly, so you must not access the element cache.

Reply:
Well, that made it a lot shorter ^^